app/javascript/application.js

```diff
@@ -1 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "trix"
+import "@rails/actiontext"
+import "./trix-editor-overrides"
```

Comment:
please end **all** files with newlines. otherwise it generates useless changes, when a the file get modified with an editor that adds those automatically during save